api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md

```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
+
+        <!--JavaScript-->customizeSeries: function (seriesName, seriesOptions) {
+            // Change series properties here
+            return seriesOptions; // This line is optional
+        }
+
+    The **seriesName** argument is a string that allows you to identify a series. The **seriesOptions** argument is an object with the following structure:
+
+    <table class="dx-table full-width">
+        <tr>
+            <th>Field name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td><b>axis</b></td>
+            <td>String</td>
+            <td>The name of the axis to which the series belongs. Has a value only when the chart has multiple axes.</td>
+        </tr>
+        <tr>
+            <td><b>pane</b></td>
+            <td>String</td>
+            <td>The name of the pane the series occupies. Has a value only when the chart has multiple panes.</td>
+        </tr>
+    </table>
+
+    You can also specify any other [series properties](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) in this object, except [seriesTemplate](Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/).
+
+- **processCell**
+
+        <!--JavaScript-->processCell: function (cellData) {
+            // Process data here
+            return cellData; // This line is optional
+        }
+        
+    The **cellData** object has the following structure:
+
+    <table class="dx-table full-width">
+        <tr>
+            <th>Field name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td><b>rowPath</b></td>
+            <td>Array</td>
+            <td>The path to the data cell by row field values.</td>
+        </tr>
+        <tr>
+            <td><b>rowPathFormatted</b></td>
+            <td>Array</td>
+            <td>Same as <b>rowPath</b>, but with <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#format" target="_blank">format</a> and <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#customizeText" target="_blank">customizeText</a> applied.</td>
+        </tr>
+        <tr>
+            <td><b>rowFields</b></td>
+            <td>Array</td>
+            <td>Contains objects that are row <a href="/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/" target="_blank">fields</a>.</td>
+        </tr>
+        <tr>
+            <td><b>maxRowLevel</b></td>
+            <td>Number</td>
+            <td>A zero-based index that indicates the maximum currently expanded row level. Equals to 0 when the pivot grid has no row fields, 1 when all rows are collapsed, 2 when at least one first-level row is expanded, etc.</td>
+        </tr>
+        <tr>
+            <td><b>columnPath</b></td>
+            <td>Array</td>
+            <td>The path to the data cell by column field values.</td>
```

Comment:
See my similar comment above. Do you mean the following?

`The path to the data cell by column field value.`